clients/imodels-client-authoring/CHANGELOG.md

```diff
@@ -1,31 +1,47 @@
-# Change Log - @itwin/imodels-client-authoring 
+# Change Log - @itwin/imodels-client-authoring
+
+This log was last generated on Wed, 29 May 2024 06:49:53 GMT and should not be manually modified.
 
 ## 5.0.0
+Thu, 4 Apr 2024 13:39:29 GMT
+
+### Breaking changes
 
-Breaking changes:
-- Updated `RestClient` interface. Classes implementing `RestClient` interface must now return HTTP response headers in addition to the response body.
-- Updated `createFromTemplate` iModel operation error handling. If iModel initialization times out, the error code of the thrown error will now be `IModelFromTemplateInitializationTimedOut` instead of `IModelFromTemplateInitializationFailed`.
+- Update `RestClient` interface. Classes implementing `RestClient` interface must now return HTTP response headers in addition to the response body.
+- Update `createFromTemplate` iModel operation error handling. If iModel initialization times out, the error code of the thrown error will now be `IModelFromTemplateInitializationTimedOut` instead of `IModelFromTemplateInitializationFailed`.
 
 ## 4.0.0
+Tue, 23 May 2023 15:57:54 GMT
+
+### Breaking changes
 
-Breaking changes:
-- Dropped support for Node.js versions older than 18.12.0.
+- Drop support for Node.js versions older than 18.12.0.
 
 ## 3.0.0
+Thu, 23 Feb 2023 16:27:30 GMT
 
-Breaking changes:
-- Updated client to use [iModels API V2](https://developer.bentley.com/apis/imodels-v2/overview/) by default. All references to the term "Project" were updated to use "iTwin" (`projectId` -> `iTwinId`, ...).
+### Breaking changes
+
+- Update client to use [iModels API V2](https://developer.bentley.com/apis/imodels-v2/overview/) by default. All references to the term 'Project' were updated to use 'iTwin' (`projectId` -> `iTwinId`, ...).
 
 ## 2.0.0
+Fri, 22 July 2022 14:42:36 GMT
+
+### Breaking changes
 
-Breaking changes:
-- Removed `FileHandler` interface and default `AzureFileHandler` implementation. Storage operations now use [`@itwin/object-storage-core`](https://www.npmjs.com/package/@itwin/object-storage-core) and [`@itwin/object-storage-azure`](https://www.npmjs.com/package/@itwin/object-storage-azure) packages.
```

Comment:
nitpick: this was lost in the new changelog

Reply:
added, thanks